src/test/kotlin/three/consulting/epoc/service/TimesheetEntryServiceIntegrationTest.kt

```diff
@@ -308,7 +308,7 @@ class TimesheetEntryServiceIntegrationTest : IntegrationTest() {
     fun `timesheet entries csv export contains the right rows`() {
         val startDate = LocalDate.parse("2022-04-01")
         val endDate = LocalDate.parse("2022-04-01")
-        val csv = timesheetEntryService.exportToCsv(startDate, endDate)
+        val csv = timesheetEntryService.exportToCsv(startDate, endDate, null)
```

Comment:
You should also add a test where you're filtering with the email as well. Now the new functionality lies in uncovered area